kivy/_clock.pyx

```diff
@@ -248,6 +249,13 @@ cdef class CyClockBase(object):
         ev.release()
         return ev
 
+    cpdef schedule_del_safe(self, callback):
+        '''Schedule a callback. Might be called from GC and cannot be cancelled.
+
```

Comment:
It'd be nice to expand a little explaining what the problem is. E.g. something like:

```
Schedule a callback. Might be called from GC and cannot be cancelled.

It's unsafe to call various kinds of code, such as code with a lock, from a `__del__` or `__dealloc__` methods. Since Kivy's Clock uses a lock, it's generally unsafe to call from these methods. Instead, use this method, which is thread safe and `__del__` or `__dealloc__` safe, to schedule the callback in the kivy thread. It'll be executed in order after the normal events are processed. 

The callback takes no parameters and cannot be canceled.
```